schedstat.go

```diff
@@ -39,16 +53,16 @@ type Schedstat struct {
 type SchedstatCPU struct {
 	CPUNum string
 
-	RunningJiffies uint64
-	WaitingJiffies uint64
-	RunTimeslices  uint64
+	Running       time.Duration
```

Comment:
Thinking about this more, let's keep this as `RunningNanoseconds uint64`.

Reply:
guh....why?

Reply:
Because the job of this library is to provide access to the data, not interpret it.

Reply:
Should the variable names not contain "nanoseconds" then, since they might be jiffies? What if the C compiler used to build the kernel has an `unsigned long long` that's longer than 64 bits?

Reply:
Hopefully Go will support longer than uint64 by then.